events/KMS/ScheduleKeyDeletion.json

```diff
@@ -9,6 +9,35 @@
     "mitreAttackTechniques": [
         "T1485 - Data Destruction"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1561 - Disk Wipe",
+            "reason": "By scheduling the deletion of a KMS key, the adversary could render encrypted data useless, effectively wiping the disk content indirectly."
+        },
+        {
+            "technique": "T1499 - Endpoint Denial of Service",
+            "reason": "Deleting a KMS key can disrupt the availability of data, causing a denial of service on the applications relying on the encrypted data."
+        },
+        {
+            "technique": "T1531 - Account Access Removal",
+            "reason": "An adversary may remove access to encrypted data by scheduling the deletion of the KMS key, which is crucial for decryption."
```

Comment:
I agree but the account itself continues to have access to the system, just the data is not there ( because of Disk Wipe )